front/src/components/SignInForm/authorizationForm.js

```diff
@@ -1,31 +1,81 @@
-import React from 'react'
-
-const AuthorizationForm = () => {
-    return(
-        <form>
-            <fieldset className="forms__fieldset">
-                <legend className="form-item forms__legend">
-                    Вход в систему
-                </legend>
-                <input className="form-item forms__text-input bordered"
-                    type="email"
-                    placeholder=" E-mail"
-                    autoComplete="on"
-                    required
-                />
-                <input className="form-item forms__text-input bordered"
-                    type="password"
-                    placeholder=" Пароль"
-                    autoComplete="off"
-                    required
-                />
-                <input className="form-item forms__button bordered"
-                    type="submit"
-                    value="Войти"
-                />
-            </fieldset>
-        </form>
-    )
+import React, {Component} from 'react'
+
+class AuthorizationForm extends Component {
+
+    state = {
+        email: "",
+        password: "",
+        isInFocus: false
+    }
+
+    constructor (props) {
+        super (props);
+        this.handleInputChange = this.handleInputChange.bind(this);
+        this.sendInfo = this.sendInfo.bind(this);
+        this.validateInputs = this.validateInputs.bind(this);
+    }
+
+    handleInputChange (event) {
+        let target = event.target;
+        this.setState({
+            [target.name]: target.value
+        });
+    }
+
+    validateInputs() {
+        return {
+            email: this.state.email.length === 0,
```

Comment:
Look for react-form-validation or redux-form-validation npm packages. What is the most convinient.

Is it possible that email can be null?